src/textacy/corpus.py

```diff
@@ -161,7 +161,7 @@ def __init__(
         self.n_docs = 0
         self.n_sents = 0
         self.n_tokens = 0
-        if data:
+        if data is not None:
```

Comment:
Is this a v3-specific issue, or just an edge case bug that has never been properly handled? Could you give a mini-example showing how this failed for you?

Reply:
Update: I'm guessing from the commit message that you added this for numpy's sake, e.g.

```
In [1]: import numpy as np

In [2]: arr = np.ndarray([1,2,3])

In [3]: if arr: print("okay")
---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
<ipython-input-3-a19c660a75e5> in <module>
----> 1 if arr: print("okay")

ValueError: The truth value of an array with more than one element is ambiguous. Use a.any() or a.all()
```

Which makes this an edge case that I never properly handled. I'm going to just add it to the latest `develop` branch, which is now working (with minimal hacks...) for spacy v3, since the other two changes in this PR were incidentally handled on the way to making the code run and tests pass.